modules/phase_field/src/postprocessors/FeatureFloodCount.C

```diff
@@ -867,16 +850,10 @@ FeatureFloodCount::calculateBubbleVolumes()
     // For each of the _maps_size FeatureData lists, determine if the set
     // of nodes includes any boundary nodes.
     for (unsigned int map_num = 0; map_num < _maps_size; ++map_num)
-    {
-      std::vector<MooseSharedPointer<FeatureData> >::iterator
-        bubble_it = _feature_sets[map_num].begin(),
-        bubble_end = _feature_sets[map_num].end();
-
       // Determine boundary intersection for each FeatureData object
-      for (; bubble_it != bubble_end; ++bubble_it)
-        (*bubble_it)->_intersects_boundary = setsIntersect(all_boundary_node_ids.begin(), all_boundary_node_ids.end(),
-                                                           (*bubble_it)->_local_ids.begin(), (*bubble_it)->_local_ids.end());
-    }
+      for (const auto & feature_ptr : _feature_sets[map_num])
```

Comment:
Indentation is correct here. This is a nested loop, there just happens to be a comment in-between.